### DISCUSSION TITLE
Fast charging  mode

### DISCUSSION THREAD

**Discussion by GeneralFizi** [seerge/g-helper#3167] [General]
Hi devs! hope you're doing well. i want to know if there's "fast charging mode" like in my asus app. seems like i cant find it on 0.191?
and after installing g helper, i cant use my asus because there's constant " please restart the app or the system to  ensure the functions work properly " notification"

**Comment by seerge** [comment]
What is fast charging mode ? What exactly it does in MyASUS ?

I think laptop always charges as fast as it can't, and it's controlled by hardware anyway :)

**Reply by GeneralFizi** [reply]
https://km-ap.asus.com/uploads/PhotoLibrarys/622e0052-9d1f-4fb5-a524-35c527fab9c9/20240109153758191_Devicesettings1_EN.png

My "MyAsus" has this options on the link above, im not sure what differentiate it from Ghelper "charge once to 100%". i thought this might be fast charging mode from MyAsus. i can see my charging speed and discharging speed from Ghelper, and while plugged in it says charging at 80W, but i have 240W adapter. so is it normal to charge below your adapter max power output ?

**Reply by seerge** [reply]
@GeneralFizi i don't see any fast charging on your screenshot.

I see an option to limit charge to 80% (which you can already do), and an option just to charge to 100% one time. 

You have both of this options in G-Helper as well. Just lick ``100%`` button next to charge limit slider.

80W as a charging speed is absolutely fine, as your laptop also needs to consume power from the adapter for current operations, not only for charging.

**Reply by GeneralFizi** [reply]
ah okay i  didnt know that's how it works. thanks Seerge :)
